frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { IParcel } from 'actions/parcelsActions';
+import { Label } from 'components/common/Label';
+import './InfoSlideOut.scss';
+
+export type IParcelAttributes = {
+  /** the selected parcel information */
+  parcelInfo: IParcel | null;
+};
+
+/**
+ * Displays parcel specific information needed on the information slide out
+ * @param parcelInfo the selected parcel data
+ */
+export const ParcelAttributes: React.FC<IParcelAttributes> = ({ parcelInfo }) => {
+  let formatAssessed;
+  if (parcelInfo?.assessed !== '') {
+    formatAssessed = '$' + new Intl.NumberFormat('en-US').format(parcelInfo?.assessed as number);
+  } else {
+    formatAssessed = '';
+  }
+  return (
+    <>
+      <ListGroup>
+        <Label className="header">Parcel attributes</Label>
+        <ListGroup.Item>
+          <Label>Lot size:</Label>
+          {parcelInfo?.landArea + ' hectares'}
```

Comment:
Didn't this get updated to sq m?

Reply:
The parcel details page says hectares, I thought it was building area that was changed to square meters?

Reply:
Need to ask Bobbi

Reply:
@bbjornholt should Lot Size be in hectares or square meters?

Reply:
it's hectares 👍